master/buildbot/test/fake/fakedb.py

```diff
@@ -1655,7 +1656,7 @@ def setState(self, objectid, name, value):
 
     def atomicCreateState(self, objectid, name, thd_create_callback):
         value = thd_create_callback()
-        self.states[objectid][name] = json.dumps(value)
+        self.states[objectid][name] = json.dumps(bytes2NativeString(value))
```

Comment:
This is needed because if you do this:
```
json.dumps(b"hello world")
```

you get this:

```
TypeError: Object of type 'bytes' is not JSON serializable
```